spec/helpers/country_flag_helper_spec.rb

```diff
@@ -35,5 +35,17 @@
         expect(Raven).to have_received(:capture_message)
       end
     end
+
+    context "with country which we don't have a flag which is in ignore list" do
```

Comment:
this should be a bit more readable:
`context 'when country do not have a flag and is in the ignore list'`